lib/collection/variable.js

```diff
@@ -341,6 +346,27 @@ _.assign(Variable, /** @lends Variable */ {
             }
         },
 
+        /**
+         * A "secret" type value stores data as postman secret variables
+         */
+        secret: {
```

Comment:
Can we remove all changes from this pr except this. Essentially decoupling the type change and secret change